controls/radupload/getting-started.md

```diff
@@ -37,20 +40,29 @@ The first thing you have to do is to add a reference to the __Telerik.Windows.Ra
 	End Class
 {{endregion}}
 
-Create a folder, in which the uploaded files will be stored.
-![](images/RadUpload_GettingStarted_02.png)
+Next, create a folder, in which the uploaded files will be stored.
+
+#### Figure 2: The UserUploads folder
+
+![The UserUploads folder](images/RadUpload_GettingStarted_02.png)
 
 >Please note that the target folder should be in the same folder as the upload handler.
 
 To test the handler point your browser to the SampleUploadHandler.ashx file. You should see the following output if everything is done correctly.
-![](images/RadUpload_GettingStarted_03.png)
+
+#### Figure 3: The ouput when navigating to the handler
+
+![The ouput when navigating to the handler](images/RadUpload_GettingStarted_03.png)
 
 ## Configuring the client side
 
 The first thing you have to do on the client side is to declare a __RadUpload__ control in the appropriate UserControl.
-		
 
->In order to use __RadUpload__ control in your projects you have to add references to the following assemblies:<br/>1. __Telerik.Windows.Controls__<br/>2. __Telerik.Windows.Controls.Input__<br/>{% if site.site_name == 'WPF' %}You can find more info [here](http://www.telerik.com/help/wpf/installation-installing-controls-dependencies-wpf.html).{% endif %}{% if site.site_name == 'Silverlight' %}You can find more info [here](http://www.telerik.com/help/silverlight/installation-installing-controls-dependencies.html).{% endif %}
+>In order to use __RadUpload__ control in your projects you have to add references to the following assemblies:
```

Comment:
Can we move the required assemblies in an "Assembly References" section in the beginning of the article? As the one in the RadGridView Getting Started for example.